dev-client/src/components/icons/Icon.tsx

```diff
@@ -33,7 +34,7 @@ export type IconProps = Omit<
 > &
   NativeBaseProps & {
     size?: keyof typeof theme.components.Icon.sizes | number;
-    color?: ThemeColor | string;
+    color?: ThemeColor | ColorValue | string;
```

Comment:
This allows us to take the output of `convertColorProp` as an input here, which is more convenient for some usages.